yarn-plugin-pin-deps.js

```diff
@@ -534,7 +534,7 @@ module.exports = {
           `$0 pin-deps`,
         ],
         [
-          `Perform a "dry run" – do not apply any changes to files, but otherwise
+          `Perform a "dry run" – do not apply any changes to files, but otherwise
```

Comment:
it was a non-breaking space (character code 160, https://www.compart.com/en/unicode/U+00A0) 

![image](https://user-images.githubusercontent.com/889383/153192695-fe0461f5-0a3e-4265-a114-f9b789c3f580.png)


instead of a normal space (character code 32) :smile: 

![image](https://user-images.githubusercontent.com/889383/153192732-0ee06663-4e27-409f-9cec-29645478b500.png)